cypress/plugins/testDataProcessor.js

```diff
@@ -50,16 +55,47 @@ function testDataProcessor(configEnv) {
   ]);
 
   // Resolving the variables in the JSON
-  const resolvedFireboltCallsJson = processFireboltJson(combinedFireboltCallsJson);
+  resolvedFireboltCallsJson = processFireboltJson(combinedFireboltCallsJson);
+
+  // Resolving the variables in the SetResponse JSON
+  const resolvedSetResponseJson = processSetResponseJson(mergedSetResponseJson);
 
   // Below key names are converted into environment variables.
   return {
     fireboltCallsJson: resolvedFireboltCallsJson,
     fireboltMocksJson: combinedFireboltMocksJson,
-    setResponseJson: mergedSetResponseJson,
+    setResponseJson: resolvedSetResponseJson,
   };
 }
 
+/**
+ *  @function processSetResponseJson
+ *  processSetResponseJson function will perform following operations
+ *  - Iterate over each key in the provided JSON
+ *  - Resolve the values of setResponse params
+ *  - Return the JSON with the updated value.
+ *
+ *  @example
+ *  processSetResponseJson({'abc': {'fireboltMock': 'ACKNOWLEDGE_CHALLENGE_GRANTED', 'firstParty': 'true'}})
+ *  @returns
+ *  {'abc': {fireboltMock: {'method': 'method_name', 'response': [{"isCancelled": false, "withUi": true, "result":{}}]}}, 'firstParty': 'true'}
+ */
+function processSetResponseJson(jsonData) {
+  // Looping through json data
+  for (const key in jsonData) {
```

Comment:
if fireboltCall/fireboltMock key value not present returning key as-is right? cross check this changes are okay and inside the setResponse glue if value not parsed we need to handle it like printing the log or failing the test

Reply:
test once it is taking key name as-is when fireboltCall/fireboltMock value not present,

Reply:
if fireboltCall/fireboltMock key value is not present, the key will be returned as it is.